nodeup/pkg/model/network.go

```diff
@@ -61,6 +60,17 @@ func (b *NetworkBuilder) Build(c *fi.ModelBuilderContext) error {
 		}
 	}
 
+	if networking.AmazonVPCIPVlan != nil {
```

Comment:
I hooked downloading of the binaries into this ModelBuilder. It seems a little awkward and I'm wondering if treating the download as another asset the same as the other [cni binaries](https://github.com/kubernetes/kops/blob/5eb1fc4985c4e148620402dc1fce4788c35feb1d/upup/pkg/fi/cloudup/apply_cluster.go#L955-L962) is the right way to go. I had to modify the Archive task to support gzip archives which seems like is already handled in the asset store.